app/assets/javascripts/bulk.js

```diff
@@ -279,7 +274,7 @@ function upd_values_for_druids(upd_req_url, upd_textarea_id, row_processing_fn,
 		
 		//skip bad rows
 		if(is_invalid_row_fn(upd_info)) {
-			log.innerHTML = "<span class=\"error\"> "+job_count.pop()+" "+object_link+" : "+invalid_row_err_msg+" '"+upd_info['upd_data']+"'</span><br>\n"+log.innerHTML;
+			log.innerHTML = "<span class=\"text-danger\"> "+job_count.pop()+" "+object_link+" : "+invalid_row_err_msg+" '"+upd_info['upd_data']+"'</span><br>\n"+log.innerHTML;
```

Comment:
:+1: for consistency, but seems like `<br/>` is a holdover from XHTML, no? Though I'm sure I've committed code with it in the past year :)